### ISSUE TITLE
nvim 0.10.2: lua error on yank: attempt to index field 'hl' (a nil value)

### ISSUE THREAD

**Issue by phpwutz** [NormalNvim/NormalNvim#559]
Hi everyone!

I'm currently on  neovim (0.10.2) /normalvim + arch linux and get this error when yanking:

```
Error detected while processing TextYankPost Autocommands for "*":
Error executing lua callback: /home/user/.config/nvim/lua/plugins/2-ui.lua:694: attempt to index field 'hl' (a nil value)
stack traceback:
        /home/user/.config/nvim/lua/plugins/2-ui.lua:694: in function </home/user/.config/nvim/lua/plugins/2-ui.lua:694>
```

it seems the culprit is in `2-ui.lua` where the 'hl' attribute of the vim object is not defined. 
I currently managed to get rid of the error by commenting out the whole "tzachar/highlight-undo.nvim" block, but it would be nice if the default distro wouldn't produce any errors like this.

best regards

**Comment by phpwutz** [comment]
I found out the hl property is only available in nvim 0.11 - so either the version requirement in the README should be updated or use the old highlight property instead. any objections? if not i'd prep a PR

**Comment by Zeioth** [comment]
@phpwutz thanks for reporting, shipping fix rn.

**Comment by Zeioth** [comment]
I can repro
![screenshot_2024-11-11_20-12-09_093980121](https://github.com/user-attachments/assets/08efa09e-88be-4fdf-b3ff-0022ea861df5)

**Comment by Zeioth** [comment]
Fix released in v4.0.2!
https://github.com/NormalNvim/NormalNvim/releases/tag/v4.0.2
